js/indicia.datacomponents/jquery.idc.filterSummary.js

```diff
@@ -193,6 +193,47 @@
         });
       }
 
+      // es-filter-param controls
+      if ($('.es-filter-param').length) {
```

Comment:
I think the outer if is unnecessary as the each statement below will just encounter a zero length array on the jQuery selector.